src/lib/cometvisu-client-openhab.js

```diff
@@ -0,0 +1,112 @@
+/* cometvisu.js (c) 2010 by Christian Mayer [CometVisu at ChristianMayer dot de]
```

Comment:
I really think we should unify that code - but at the moment I don't know how, though...

So I'll merge it now and we can try to make a unification later